replication-api/src/main/java/org/codice/ditto/replication/api/data/Persistable.java

```diff
@@ -36,4 +38,14 @@
    * @return integer version of the persistable
    */
   int getVersion();
+
+  /**
+   * Returns a {@link Date} at which the persistable was last modified
+   *
+   * @return when the persistable was last modified
+   */
+  Date getModified();
+
+  /** Sets the modified date of this persistable to the current time */
+  void setModified();
```

Comment:
✏️ I don't think we should expose this method. Instead, I would make sure we document the `save()` to indicate it will automatically update that field.